webgpu/lessons/webgpu-orthographic-projection.md

```diff
@@ -377,8 +377,8 @@ work them out you'd see them simplify just like before
 Z rotation
 
 <div class="webgpu_center"><pre class="webgpu_math">
-newX = x *  c + y * s;
-newY = x * -s + y * c;
+newX = x * c + y * -s;
+newY = x * s + y *  c;
```

Comment:
This now matches its corresponding matrix ilustration. And checking:

Rotating +X (1,0,0) by 90° gives +Y (0,1,0).
Rotating +Y by 90° gives -X. Looks good.